docker.go

```diff
@@ -79,16 +94,36 @@ func (container *Container) Volume(hostPath, containerPath, mode string) *Contai
 func NewDockerContainer() *Container {
 	container := new(Container)
 	container.Type = "DOCKER"
-	container.Docker = &Docker{
-		Image:        "",
-		Network:      "BRIDGE",
-		PortMappings: make([]*PortMapping, 0),
-		Parameters:   make([]*Parameters, 0),
-	}
-	container.Volumes = make([]*Volume, 0)
+	container.EmptyVolumes()
+	container.Docker = &Docker{}
+	container.Docker.Container("").
+		Bridged().
+		SetPrivileged(false).
```

Comment:
mmm ... I think there are valid use cases for both scenarios:
  - I want to have everything empty and overwrite with empty values vs
  - I want to get an empty structure for updating and will only set values that I want to update
... the latter use case can be achieved by fetching the app from marathon before hand, the first one would mean that I manually have to set zero-values for all properties. If there will be a new property, lib users would have to adjust their code.